sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSinkSuite.scala

```diff
@@ -274,6 +277,7 @@ class FileStreamSinkSuite extends StreamTest {
             "CAST(start as BIGINT) AS start",
             "CAST(end as BIGINT) AS end",
             "count")
+          .orderBy("start") // sort the DataFrame in order to compare with the expected one.
```

Comment:
Talked with @jose-torres offline. The ordering of result here is not guaranteed. Add `orderBy` to fix the test case.